crates/network-scanner/Cargo.toml

```diff
@@ -8,12 +8,20 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+crossbeam = { version = "0.8.4", features = ["crossbeam-channel"] }
+dashmap = "5.5.3"
+derive_builder = "0.13.0"
+futures = { version = "0.3.30" }
+lazy_static = "1.4.0"
+network-interface = "1.1.1"
 network-scanner-net ={ path = "../network-scanner-net" }
 network-scanner-proto ={ path = "../network-scanner-proto" }
+serde = { version = "1.0.196", features = ["serde_derive"] }
 socket2 = "0.5.5"
-tokio = { version = "1.36.0", features = ["io-util"] }
+thiserror = "1.0.56"
+tokio = { version = "1.35.1" , features = ["rt"]}
 tracing = "0.1.40"
 
 [dev-dependencies]
-tokio = { version = "1.36.0", features = ["rt","macros","rt-multi-thread"] }
+tokio = { version = "1.35.1", features = ["rt","macros","rt-multi-thread","tracing"] }
```

Comment:
issue: Probably a merge conflict which went wrong, but we are now using tokio 1.36.0, not 1.35.1

Reply:
updated